### PR TITLE
skip attempt or batch if ci skip is present in patch description

### PR DESCRIPTION
**Pull request by khodzha**
#252

### CODE REVIEW

**Comment by khodzha** [comment]
not sure how i should test skipping batch/attempt run

**Comment by khodzha** [comment]
i think i accidentally skipped ci :rofl:

File: apps/bors_database/test/models/patch_test.exs [hunk 1]
```diff
@@ -207,4 +207,24 @@ defmodule BorsNG.Database.PatchTest do
...
+
+  test "ci_skip? checks title" do
+    p = %Patch{
+      pr_xref: 9,
+      title: "[ci skip] title",
+      body: "this is body",
+      commit: "C"
+    }
+    assert Patch.ci_skip?(p)
+  end
```
Comment (notriddle) [inline apps/bors_database/test/models/patch_test.exs]: This is conspicuously missing a negative test.

**Comment by notriddle** [comment]
Since the PR message itself doesn't contain `[ci skip]`, it should still work when I r+ it.

**Pull request merged**